app/lib/apply_api/client.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module ApplyApi
+  class Client
+    include HTTParty
+    # TODO: Make it real!
+    base_uri Settings.apply.api_base_url
+    headers "Accept" => "application/json",
+            "Content-Type" => "application/json;odata.metadata=minimal",
```

Comment:
I admit I don't know what Apply will do with `odata.metadata=minimal` — happy to add support if required!

Reply:
Not required! Copy pasta from our DTTP client, I'll remove.